### ISSUE TITLE
(TSQL) STRING_AGG is missing from Lexer/Parser (FIX INCLUDED)

### ISSUE THREAD

**Issue by Szhlopp** [antlr/grammars-v4#1597]
The below script does not run currently because the `STRING_AGG` function isn't included in the g4 files. Adding the proper Lexer token and addition to the `function_call` rule will allow `STRING_AGG` scripts to be parsed properly.

```g4
STRING_AGG:                            'STRING_AGG';

function_call
| STRING_AGG '(' aggcol=expression ',' (expression | STRING) ')' (WITHIN GROUP '(' order_by_clause ')')?  #STRINGAGG
```

```sql
SELECT
	ETLTYPE,
	STRING_AGG(ETLNAME, ',') WITHIN GROUP (ORDER BY TABLENAME) AS NAMES
FROM CONFIG.ETL
GROUP BY
	ETLTYPE
```

**Comment by KvanTTT** [comment]
Could you please suggest a pull request with your fix?

**Comment by Szhlopp** [comment]
@KvanTTT I've created a pull request with the two issues I've discovered. Please let me know if you need anything else. 

Thanks!

**Comment by KvanTTT** [comment]
fixed by #1598
